js/client/modules/@arangodb/testsuites/replication.js

```diff
@@ -529,7 +529,7 @@ function replicationSync (options) {
       let shutdownState = true;
       let res = true;
       print("starting replication slave: ");
-      let slave = pu.startInstance('tcp', options, {"log.level" : "replication=trace", "--log.level": "replication=trace"}, 'slave_sync');
+      let slave = pu.startInstance('tcp', options, {"log.level" : "replication=info", "--log.level": "replication=info"}, 'slave_sync');
```

Comment:
please make this configurable by `if (options.extremeVerbosity)`

Reply:
None of the other replication testsuites has something like this.
I remember we added this a long time ago when debugging some replication issues, and never removed it later.
As I said, there are numerous other replication suites which all don't have this, and my position is that we should remove it here too. It is just a leftover from long-ago debugging.

Reply:
`extremeVerbosity` has to be specified from the commandline. So it will not be there in jenkins runs. 
If you specify it, you want it noisy.

Reply:
I understand, but why should we honor the flag here, in this single testsuite, but in none of the other replication testsuites?
I think we don't need this at all.
It was there as a leftover from debugging a specific problem, which has been solved long ago.